bootstrap.js

```diff
@@ -17,21 +21,28 @@ module.exports = function(app){
     app.get('/*', (req, res) => {
         let origionalUrl = req.originalUrl;
         let requestedUrl = req.params[0];
+        let reqUrl = url.parse(requestedUrl)
```

Comment:
reqUrl isn't very informative or expressive to what this variable actually is, I think a better name would be formattedRequestUrl   .